README.md

```diff
@@ -42,6 +42,7 @@ jobs:
       - name: Run yor action
         uses: bridgecrewio/yor-action@main
```

Comment:
Recommending to use `@main` is dangerous IMO and is why I felt like I could only be backward compatible.
Github guidelines are followed by most actions: https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-release-management-for-actions